src/checkbox.scss

```diff
@@ -6,6 +6,7 @@
 .fd-checkbox
 */
 $block: #{$fd-namespace}-checkbox;
+$displayonlyblock: #{$fd-namespace}-displayonly-checkbox;
```

Comment:
I don't think this should be a component. Display-only mode should be a modifier class. Example:
`fd-checkbox--display-only`